src/App.js

```diff
@@ -1,14 +1,36 @@
 import React from "react";
 import "./App.css";
+import Increment from "./src./Increment";
+// import Score from "./Score";
+// import playAgain from ".playAgain";
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      currentScore: 0,
+    };
+  }
+
+  //event handler
+  changeScore = () => {
+    this.setState({ currentScore: this.state.currentScore + 1 });
+  };
   render() {
     return (
       <main>
-        <p>React State Lab</p>
+        <h2>Current Score: {currentScore}</h2>
       </main>
+      {if(currentScore >= 10){
```

Comment:
This if statement is breaking your code. You need to figure out another way to use a conditional. Maybe consider moving the conditional outside of the return.